packages/alias/test/test.js

```diff
@@ -10,6 +10,47 @@ import alias from '../dist';
 const normalizePath = (pathToNormalize) => slash(pathToNormalize.replace(/^([A-Z]:)/, ''));
 const DIRNAME = normalizePath(__dirname);
 
+// "plugins" is an array of plugins for Rollup, they should include "alias"
+// "tests" is an array of pairs [importee, importer]
+function resolveWithRollup(plugins, tests) {
```

Comment:
if `plugins` should include `alias` plugin - you could verify that at runtime to avoid misusage in the future
```js
if (!plugins.find(p => p.name === 'alias')) {
  throw new Error('`plugins` should include the alias plugin.')
}
```